packages/@react-aria/tag/src/useTag.ts

```diff
@@ -56,7 +56,7 @@ export function useTag(props: TagProps<any>, state: GridState<any, any>): TagAri
   }, state, tagRef);
 
   function onKeyDown(e: KeyboardEvent<HTMLElement>) {
-    if (e.key === 'Delete' || e.key === 'Backspace') {
+    if (e.key === 'Delete' || e.key === 'Backspace' || e.key === 'Escape' || e.key === ' ') {
```

Comment:
It looks like some wires got crossed. I talked to @LFDanLu about this behavior. What we want is only Delete, Backspace, and Space as per the Spectrum Guidelines. Can you add Space to the unit tests?
```suggestion
    if (e.key === 'Delete' || e.key === 'Backspace' || e.key === ' ') {
```